core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -29,6 +29,8 @@ private CatalogProperties() {
   public static final String CATALOG_IMPL = "catalog-impl";
   public static final String FILE_IO_IMPL = "io-impl";
   public static final String WAREHOUSE_LOCATION = "warehouse";
+  public static final String TABLE_DEFAULT_PREFIX = "table-default.";
+  public static final String TABLE_OVERRIDE_PREFIX = "table-override.";
```

Comment:
Nit / open-question: How do people feel about these names? I'd kind of like them to have `properties` or something in them, but this might be overkill.

Reply:
I'm good with this.